pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -568,13 +568,13 @@ func (k *Bootstrapper) needsReconfigure(conf string, hostname string, port int,
 		klog.Infof("needs reconfigure: configs differ:\n%s", rr.Output())
 		return true
 	}
-
-	st, err := kverify.APIServerStatus(k.c, hostname, port)
+	// cruntime.Enable() may restart kube-apiserver but does not wait for it to return back
+	apiStatusTimeout := 1500 * time.Millisecond
```

Comment:
since this var is not used anywhere else why not pass it directly ?

Reply:
First, it makes the next line shorter and easier to read. Another reason why I like this style is that a variable name gives a bit more context about the value, working like a comment. In this case variable's purpose is more or less obvious, but in general such approach helps to understand the meaning of parameters without looking into the target function.